src/System.IO.IsolatedStorage/tests/System/IO/IsolatedStorage/GetCreationTimeTests.cs

```diff
@@ -49,9 +49,9 @@ public void GetCreationTime_Closed_ThrowsInvalidOperationException()
             }
         }
 
-        [ActiveIssue(25665)]
-        [Fact]
-        public void GetCreationTime_RaisesArgumentException()
+        [Fact] 
+        [ActiveIssue(25665, ~TestPlatforms.Windows)]
+        public void GetCreationTime_RaisesArgumentException_Windows()
```

Comment:
Is the intention that this will run on non-windows OSes?  If so, why does the name include _Windows?

Reply:
It will run on Non-Windows. But the function body will be quite different. So I decided to add a new test with name GetCreationTime_RaisesArgumentException_Unix (In another PR).
Should I revert back this change for this PR ?